### PR TITLE
Benchmarking Deviated Wellbore problems

### PR DESCRIPTION
**Pull request by sytuannguyen**
This PR aims to add xml input files and python scripts for benchmarking deviated wellbore problems. The most relevant references ([Abousleiman and Cui, 1998](https://github.com/GEOSX/GEOSX/files/6413687/abousleiman1998.pdf), [Cui et al., 1997](https://github.com/GEOSX/GEOSX/files/6413695/cui1997.pdf)) will be considered to validate GEOSX results.

### CODE REVIEW

**Comment by sytuannguyen** [comment]
Now mesh for highly deviated wellbore can be correctly generated

image

**Comment by sytuannguyen** [comment]
Now mesh for highly deviated wellbore can be correctly generated

image

**Comment by sytuannguyen** [comment]
image

**Comment by sytuannguyen** [comment]
image

**Comment by sytuannguyen** [comment]
image

**Comment by sytuannguyen** [comment]
image

**Comment by sytuannguyen** [comment]
Benchmark injecton problem in deviated poroelastic wellbore

image

image

File: src/coreComponents/physicsSolvers/multiphysics/benchmarks/poroelasticDeviatedWellbore.xml [hunk 1]
```diff
@@ -0,0 +1,272 @@
...
+      objectPath="nodeManager"
+      fieldName="TotalDisplacement"
+      component="2"
+      scale="0.0"
+      setNames="{ zneg }"/>
+
+    <!--Traction
+      name="Sx"
+      objectPath="faceManager"
+      tractionType="1"
```
Comment (sytuannguyen) [inline src/coreComponents/physicsSolvers/multiphysics/benchmarks/poroelasticDeviatedWellbore.xml:218]:
```suggestion
      tractionType="normal"
```

File: src/coreComponents/physicsSolvers/multiphysics/benchmarks/poroelasticDeviatedWellbore.xml [hunk 2]
```diff
@@ -0,0 +1,272 @@
...
+      scale="-20.0e6"
+      setNames="{ xpos }"/-->
+
+    <!--Traction
+      name="Sy"
+      objectPath="faceManager"
+      tractionType="1"
```
Comment (sytuannguyen) [inline src/coreComponents/physicsSolvers/multiphysics/benchmarks/poroelasticDeviatedWellbore.xml:225]:
```suggestion
      tractionType="normal"
```

File: src/coreComponents/physicsSolvers/multiphysics/benchmarks/poroelasticDeviatedWellbore.xml [hunk 3]
```diff
@@ -0,0 +1,272 @@
...
+      scale="-20.0e6"
+      setNames="{ ypos }"/-->
+
+    <!--Traction
+      name="Sz"
+      objectPath="faceManager"
+      tractionType="1"
```
Comment (sytuannguyen) [inline src/coreComponents/physicsSolvers/multiphysics/benchmarks/poroelasticDeviatedWellbore.xml:232]:
```suggestion
      tractionType="normal"
```

File: src/coreComponents/physicsSolvers/solidMechanics/benchmarks/DeviatedElasticWellbore.xml [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+      objectPath="nodeManager"
+      fieldName="TotalDisplacement"
+      component="2"
+      scale="0.0"
+      setNames="{ zneg }"/>
+
+    <Traction
+      name="innerPressure"
+      objectPath="faceManager"
+      tractionType="1"
```
Comment (sytuannguyen) [inline src/coreComponents/physicsSolvers/solidMechanics/benchmarks/DeviatedElasticWellbore.xml:132]:
```suggestion
      tractionType="normal"
```

**Review by sytuannguyen — COMMENTED** [review]
update tractionType keywords

**Comment by sytuannguyen** [comment]
Benchmark a drilling problem of an elastic deviated wellbore considering in-situ stresses and mud pressure

image

**Comment by sytuannguyen** [comment]
Drilling problem of a Poroelastic deviated wellbore considering in-situ stresses and pore pressure

image

**Comment by jhuang2601** [comment]
@sytuannguyen Can you prepare a rst file for this one and add it to the advancedExample folder?

**Comment by sytuannguyen** [comment]
This PR is now replaced by #1571 and #1585

**Pull request closed without merging**